src/npctalk.cpp

```diff
@@ -1533,6 +1542,14 @@ std::string dialogue::dynamic_line( const std::string &topic ) const
     } else if( topic == "TALK_DEMAND_LEAVE" ) {
         return _("Now get out of here, before I kill you.");
 
+    } else if( topic == "TALK_SHOUT" ) {
+        alpha->shout();
```

Comment:
This doesn't consume moves, while NPC-less shouting does.
Shouting should probably be free, it doesn't take much time IRL.